pkg/networkmanager/ip-settings.jsx

```diff
@@ -316,7 +316,7 @@ export const IpSettingsDialog = ({ topic, connection, dev, settings }) => {
                                 <Switch
                                     isChecked={!ignoreAutoRoutes}
                                     isDisabled={!canAuto}
-                                    onChange={value => setIgnoreAutoRoutes(!value)}
+                                    onChange={(_event, value) => setIgnoreAutoRoutes(!value)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230520-070544-54b4f0d5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)